instances/service/AllenInstituteCellTypesDataPortal.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/AllenInstituteCelTypesDataPortal",
```

Comment:
you miss a L in "cell" in "AllenInstituteCelTypesDataPortal"

Reply:
fixed